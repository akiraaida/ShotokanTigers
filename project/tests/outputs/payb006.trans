10 Ashe Ace             00000 00000.00 S 
03 Ashe Ace             12348 00010.00 TV
01 Ashe Ace             12348 00000.10   
03 Ashe Ace             12348 01995.00 CQ
01 Ashe Ace             12348 00000.10   
03 Ashe Ace             12355 01995.00 TV
01 Ashe Ace             12355 00000.10   
03 Ashe Ace             12348 01990.00 TV
01 Ashe Ace             12348 00000.10   
03 Ashe Ace             12348 00005.00 CQ
01 Ashe Ace             12348 00000.10   
03 Ashe Ace             12355 00005.00 TV
01 Ashe Ace             12355 00000.10   
03 Ashe Ace             12355 02000.00 CQ
01 Ashe Ace             12355 00000.10   
00                      00000 00000.00   
