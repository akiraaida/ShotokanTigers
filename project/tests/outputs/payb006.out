Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
9999.90
Daily Payment Limit to TV Remaining (12348):
1990.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
8004.80
Daily Payment Limit to CQ Remaining (12348):
5.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12355):
8004.90
Daily Payment Limit to TV Remaining (12355):
5.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
6014.70
Daily Payment Limit to TV Remaining (12348):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
6009.60
Daily Payment Limit to CQ Remaining (12348):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12355):
7999.80
Daily Payment Limit to TV Remaining (12355):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12355):
5999.70
Daily Payment Limit to CQ Remaining (12355):
0.00
Please enter a command: 
